### PR TITLE
Updates to seminar pages

### PR DESCRIPTION
**Pull request by kmnip**
- update index page to point to January seminar page
- add links to RSVP forms for January~April seminars
- add January seminar trainee speaker talk title
- other minor edits

### CODE REVIEW

File: src/archive/2023/2023-04-20.md [hunk 1]
```diff
@@ -8,6 +8,10 @@
 
     :material-map-marker: **Location:** St. Paul's Hospital ([1081 Burrard Street, Vancouver, BC V6Z 1Y6](https://goo.gl/maps/uCFbWCXcVrLmMnch7)), Cullen Family Lecture Theatre ([Providence Building, Level 1, Room 1477](https://docs.google.com/document/d/1xHHd14LcrDIZLG7RGBGneLgf12H-FJwpyH7rotQCo9k/edit?usp=sharing))
 
+    **RSVP**
+
+    If you are interested in attending this seminar, please fill out [the RSVP form](https://forms.gle/FZTes9e5gJcapAUC7).
```
Comment (cmrn-rhi) [inline src/archive/2023/2023-04-20.md:13]: The form on the other end of this URL specifies "VanBUG seminar - Thursday, January 19th @ 6:00 PM" when it should be April info. It's a different link from the January file so I assume it just needs to be updated on the other end @kmnip ?

**Review by cmrn-rhi — DISMISSED** [review]
All good, just need to update the information on the April RVSP form.

File: src/archive/2023/2023-04-20.md (hunk 1 above)
Comment (kmnip) [reply src/archive/2023/2023-04-20.md]: Thanks for catching this! It is fixed now!

**Comment by kmnip** [comment]
I have updated the form info and the link.
Not sure why Google would create a different short URL, but I realized the old URL actually still works.
So, this PR (the new commit) needs to be reviewed again...

**Review by cmrn-rhi — APPROVED** [review]

**Pull request merged**